packages/synapse-react-client/src/components/DataGrid/columns/AutocompleteColumn.tsx

```diff
@@ -39,33 +40,47 @@ export function AutocompleteCell({
   stopEditing,
   active,
 }: AutocompleteCellProps) {
-  const ref = useRef<HTMLInputElement>(null)
+  const [menuIsOpen, setMenuIsOpen] = useState(false)
+  const textInputRef = useRef<HTMLInputElement>(null)
+
+  const rowDataAsString = castCellValueToString(rowData)
+  const [localInputState, setLocalInputState] =
+    useState<string>(rowDataAsString)
 
-  useLayoutEffect(() => {
-    if (focus) {
-      ref.current?.focus()
+  useEffect(() => {
```

Comment:
Attempting to keep the open/focus state synchronized with `active`/`focus` props seems to work slightly better